### DISCUSSION TITLE
Building OpenTCS 5.17.0 with VDA5050 0.20.0

### DISCUSSION THREAD

**Discussion by sajalgupta0808** [openTCS/opentcs#157] [General]
Hi, I am trying to build a project with OpenTCS 5.17 or OpenTCS5.17.1 with VDA5050 0.20.0 
I am able to build this project successfully using OpenTCS Integration Example by changing the versions in dependencies but i want to build it using code of both OpenTCS 5.17 and VDA 0.20.0 

Can someone help me or guide me please, what changes should i do.

**Comment by martingr** [comment]
The [opentcs-integration-example](https://github.com/openTCS/opentcs-integration-example) project has been updated to openTCS 6.0 and version 0.21.0 of the VDA5050 vehicle driver. I recommend to create a new integration project skeleton by executing the `cloneProject` Gradle task as described in the opentcs-integration-example project's README file. With the update to openTCS 6.0, note that the project now uses Java 21.
